pkg/controller/node/status.go

```diff
@@ -27,14 +33,30 @@ func (ctrl *Controller) syncStatusOnly(pool *mcfgv1.MachineConfigPool) error {
 		return err
 	}
 
-	newStatus := calculateStatus(cc, pool, nodes)
+	machineConfigStates := []*mcfgalphav1.MachineConfigNode{}
+	ms := &mcfgalphav1.MachineConfigNode{}
+	fg, err := ctrl.fgAcessor.CurrentFeatureGates()
+	if err != nil {
+		klog.Errorf("Could not get FG: %w", err)
+	} else if fg.Enabled(configv1.FeatureGateMachineConfigNodes) {
+		for _, node := range nodes {
+			ms, err = ctrl.client.MachineconfigurationV1alpha1().MachineConfigNodes().Get(context.TODO(), node.Name, metav1.GetOptions{})
```

Comment:
Would we be able to get this off a lister?

Reply:
maybe? i feel like this may introduce some weird races.  How up to date are the listers?